### ISSUE TITLE
Repo version of Quod Libet does not open

### ISSUE THREAD

**Issue by dtalon42** [quodlibet/quodlibet#4514]
Steps to reproduce
------------------
- Tried to open Quod Libet

Expected Output
---------------
- Quod Libet should open

Actual Output
-------------
- Running quodlibet in the terminal resulted in it not opening and returning the following message:
- - `/usr/bin/python3: undefined symbol: unity_launcher_entry_get_for_desktop_id](errorreport.main.errorhook: unity.py:46:init: gi.repository.GLib.GError: g-invoke-error-quark: Could not locate unity_launcher_entry_get_for_desktop_id: 'unity_launcher_entry_get_for_desktop_id': /usr/bin/python3: undefined symbol: unity_launcher_entry_get_for_desktop_id (1)`
- Running with sudo returned this:
- - `error: failed to open file /usr/lib64/lv2/svg/manifest.ttl (No such file or directory)
lilv_world_load_file(): error: Error loading file file:///usr/lib64/lv2/svg/manifest.ttl
lilv_world_load_bundle(): error: Error reading file:///usr/lib64/lv2/svg/manifest.ttl
error: failed to open file /usr/lib64/lv2/valve_audio_dsp_version.txt/manifest.ttl (Not a directory)
lilv_world_load_file(): error: Error loading file file:///usr/lib64/lv2/valve_audio_dsp_version.txt/manifest.ttl
lilv_world_load_bundle(): error: Error reading file:///usr/lib64/lv2/valve_audio_dsp_version.txt/manifest.ttl
E: [0000.890] errorreport.main.errorhook: unity.py:46:init: gi.repository.GLib.GError: g-invoke-error-quark: Could not locate unity_launcher_entry_get_for_desktop_id: 'unity_launcher_entry_get_for_desktop_id': /usr/bin/python3: undefined symbol: unity_launcher_entry_get_for_desktop_id (1)`

Test System
-----------

### Which version of Quod Libet?
- Quod Libet 4.6

### Which operating system 
- Nobara 39
- - Basically Fedora 39

### If it's audio-related, what back-end?

Additional Information
----------------------
Flatpak version does open without issue but I wanted to try this since I had a few issues with it occasionally playing the wrong track or crashing on loading my library.
